src/components/settings-row/settings-row.component.js

```diff
@@ -17,6 +17,7 @@ const marginPropTypes = filterStyledSystemMarginProps(
 
 const SettingsRow = ({
   title,
+  headingType = "h1",
```

Comment:
I actually made a note in the Jira ticket to say that as well as allowing consumers to override the heading type, I thought we should change the default from `h1`. Admittedly that's just my opinion, and I recognise that it might technically be a "breaking change" to update this. But I don't think it's actually breaking in practice because I don't see how there are *any* legitimate uses of `SettingsRow` where an h1 is remotely sensible.

I think there are advantages in making the default into, say, `h4` (happy for you to consult on the best option), because even though this could still result in incorrect heading levels if consumers don't update it, it's likely to be a much less serious problem for screenreader users than if there's a random h1 in an unexpected place.